.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: false
+sudo: required
```

Comment:
It seems builds were working fine using the container-based builds. Why do we need to opt-out of that?